### DISCUSSION TITLE
Discussion: Nitro on Steam Deck with Jan hitting it

### DISCUSSION THREAD

**Discussion by hwc0x01** [janhq/nitro#422] [Show and tell]
Yes. You read that right. I am running a nitro server on my steam deck. I've attached a text file with how I got it running here. 
[Nitro on Steam Deck.txt](https://github.com/janhq/nitro/files/14062063/Nitro.on.Steam.Deck.txt)

**To Reproduce**
Do all of the stuff in the txt then start nitro on 0.0.0.0:3928 and you can hit it with jan by using the steps in the jan docs
if you just want to hit it locally, you can run it the normal way with just nitro

**Screenshots**
![image](https://github.com/janhq/nitro/assets/54515725/c938c517-0f75-4a7d-b54a-2c8da368f010)
![image](https://github.com/janhq/nitro/assets/54515725/1875e8d2-6a2d-4e16-86eb-9d609c2cde30)
![image](https://github.com/janhq/nitro/assets/54515725/6ef34b46-cd5c-42df-aee8-831b6be7a2ec)
![image](https://github.com/janhq/nitro/assets/54515725/fdfd2ee6-e575-4484-8414-14c16f72e7ce)

@hahuyhoang411

**Comment by hwc0x01** [comment]
Major note: because of steamOS locked file system all this has to get re-installed after OS updates. there are ways to make it persist but I was too lazy to do all of that while tinkering around the first time.

**Comment by tikikun** [comment]
awesome
